docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
     ports:
       - "127.0.0.1:8888:8888"
     volumes:
-      - ./notebooks:/notebooks
+      - "${NOTEBOOKS:-./notebooks}:/notebooks"
```

Comment:
I think we should support this also in `graph-notebook-test`.

Reply:
Yes makes sense and is easy.